src/frontend/src/env/types/env-oisy-metadata.ts

```diff
@@ -6,5 +6,6 @@ export const oisyMetadata = z.object({
 	OISY_ONELINER: z.string(),
 	OISY_DESCRIPTION: z.string(),
 	OISY_REPO_URL: z.string().url(),
-	OISY_ALPHA_WARNING_URL: z.string().url()
+	OISY_ALPHA_WARNING_URL: z.string().url(),
+	OISY_TWITTER_URL: z.string().url()
```

Comment:
If I would be picky I would say that the zod parser should validate the URL is indeed a Twitter URL 😉